R/package.R

```diff
@@ -1,10 +1,8 @@
-
-#' Advice on R Package Building
+#' goodpractice: Advice on R Package Building
 #'
 #' Give advice about good practices when building R packages. Advice includes
 #' functions and syntax to avoid, package structure, code complexity, code
 #' formatting, etc.
-#'
-#' @docType package
-#' @name goodpractice
-NULL
+#' @name goodpractice-package
+#' @aliases NULL
+"_PACKAGE"
```

Comment:
why _PACKAGE? Can you at least remove the alias to NULL and add back the `@docType`?